queries.yaml

```diff
@@ -189,12 +189,14 @@ post_process:
       source_layer: landuse
       label_property_name: label_placement
       label_property_value: "yes"
+      drop_keys: [ mz_is_building ]
   - fn: TileStache.Goodies.VecTiles.transform.generate_label_features
     params:
       source_layer: landuse
       new_layer_name: landuse_labels
       label_property_name: label_placement
       label_property_value: "yes"
+      drop_keys: [ mz_is_building ]
```

Comment:
D'oh. That's just me typo'ing the title of the PR, it should have been `mz_is_building`, same as the parent issue #333. As far as I know, we don't have a property called `mz_is_landuse`, but if you see something like that in the output, we can now easily clean it out :smile: